webview-ui/src/Kaito/Kaito.module.css

```diff
@@ -1,78 +1,91 @@
-.kaitoPageHeader {
-    color: var(--Base-Base-01, #fff);
-    font-size: 1.625rem;
-    font-style: normal;
-    font-weight: 600;
-    line-height: normal;
-}
-
 .subHeader {
-    width: 41rem;
+    width: 100vw;
```

Comment:
can we do width's as well in rem? just to make sure it works in all vscode themes? please